src/init.cpp

```diff
@@ -437,16 +437,17 @@ void SetupServerArgs()
     gArgs.AddArg("-torpassword=<pass>", "Tor control port password (default: empty)", false, OptionsCategory::CONNECTION);
 #ifdef USE_UPNP
 #if USE_UPNP
-    gArgs.AddArg("-upnp", "Use UPnP to map the listening port (default: 1 when listening and no -proxy)", false, OptionsCategory::CONNECTION);
+    gArgs.AddArg("-portmap", "Use UPnP to map the listening port (default: 1 when listening and no -proxy)", false, OptionsCategory::CONNECTION);
 #else
-    gArgs.AddArg("-upnp", strprintf("Use UPnP to map the listening port (default: %u)", 0), false, OptionsCategory::CONNECTION);
+    gArgs.AddArg("-portmap", strprintf("Use NAT-PMP to map the listening port (default: %u)", 0), false, OptionsCategory::CONNECTION);
 #endif
```

Comment:
Both should be supported...

Reply:
I meant to use same switch "-portmap" for upnp as well as NAT-PMP. And the particular implementation is picked based on "USE_UPNP" flag (In this file as well as net.cpp). So I believe both are supported but picked based on USE_UPNP flag.

Reply:
USE_UPNP is a compile-time option, not run-time.

UPnP and NAT-PMP are two completely different protocols. They don't do the same thing. Some routers may support only one or the other.

Reply:
@laanwj @luke-jr Do you think we should introduce a separate switch for this? 
#11902